test/reporters/base.spec.js

```diff
@@ -168,18 +170,46 @@ describe('Base reporter', function () {
     it("should truncate overly long 'actual' ", function () {
       var actual = '';
       var i = 0;
-      while (i++ < 120) {
+      while (i++ < 500) {
         actual += 'a foo unified diff ';
       }
       var expected = 'a bar unified diff';
 
       inlineDiffsStub.value(false);
       var output = generateDiff(actual, expected);
 
-      expect(output, 'to match', / \.\.\. Lines skipped/);
+      expect(output, 'to match', /output truncated/);
     });
 
     it("should truncate overly long 'expected' ", function () {
+      var actual = 'a foo unified diff';
```

Comment:
These tests don't cover the usage of `reporter-option maxDiffSize=0`. There are similar tests for the `spec` and `json` reporters, indeed terrible tests. Could you have a look and try to add one test, please?

Reply:
Yes, I guess so. I have a look tomorrow.

Reply:
Your tests are ok, `Base.maxDiffSize` gets changed/polluted for the complete test run though. I'm not happy with the entire spec file, I will try to improve it.